src/CryptoAPIs.jl

```diff
@@ -169,4 +169,4 @@ include("Okex/Okex.jl")
 include("Kucoin/Kucoin.jl")
 include("Upbit/Upbit.jl")
 
-end
+end
```

Comment:
Please do not remove empty lines at the end of files
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file